lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -13,10 +13,10 @@
   display: inline-block;
   position: relative;
   width: nx-dropdown-variables.$dropdown-width;
-}
 
-.nx-dropdown-icon {
-  color: var(--nx-swatch-grey-05);
+  .nx-icon:not(.nx-icon--colorful) {
```

Comment:
:thinking: is this technically a breaking change? If a user in the past wanted to include an icon with a specific color in and `NxDropdown`, all they had to do is _not_ provide it with the `nx-dropdown-icon` class.  With this PR, that will no longer work, they must instead provide the `.nx-icon--colorful` class. I think we need to deprecate but still support the previous way.

Reply:
> 🤔 is this technically a breaking change? If a user in the past wanted to include an icon with a specific color in and `NxDropdown`, all they had to do is _not_ provide it with the `.nx-dropdown-icon` class. With this PR, that will no longer work, they must instead provide the `.nx-icon--colorful` class. I think we need to deprecate but still support the previous way.

Updated here: 48a6e95bce989f25086e29f9dc8caccb6653530b

Reply:
> Updated here: https://github.com/sonatype/sonatype-react-shared-components/commit/48a6e95bce989f25086e29f9dc8caccb6653530b

Actually the more I think about it the more I'm not sure it's possible for this to not be a breaking change.  Previously, if the caller wanted the icon to keep its original color, they would _not_ specify a class.  And if they wanted it to get the dark color, they'd specify `.nx-dropdown-icon`.  Now, if they want it to keep its original color, they specify `.nx-icon--colorful`.  But what do they do if they want it to get the dark color? The only answer that makes sense is that they don't specify any class.  But that conflicts with the previous behavior, so I think it has to be a breaking change.  Luckily, we are close to being able to release RSC 12.0.0 so this could just be lumped in with that.